CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ Main (unreleased)
 
 - Add an `otelcol.exporter.kafka` component to send OTLP metrics, logs, and traces to Kafka.
 
+- Add a live debugging feature to the UI, including support for the `prometheus.relabel` component. (@wildum)
```

Comment:
I think this might be underselling it; can we explain what live debugging does? Something like:

> - Added "live debugging" to the UI. Live debugging streams data as they flow through components for debugging telemetry data. Individual components must be updated to support live debugging. (@wildum) 
> - Adding live debugging support for `prometheus.relabel`. (@wildum)